lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -352,6 +352,9 @@ bool isUnaryOp(linalg::GenericOp linalgOp) {
 }
 
 // Return true if the linalg.generic can be mapped to a tpp.relu.
+// TODO: check input arguments:
+// 1. Single input, single output as long as the output bbarg is not used
+// 2. Single output.
```

Comment:
Is this comment still relevant? I have pushed a similar PR here, no?

Reply:
Will remove.